build.gradle.kts

```diff
@@ -60,7 +60,7 @@ dependencies {
     detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.21.0")
     testImplementation("org.junit.jupiter:junit-jupiter:5.8.2")
 
-    implementation("com.cognifide.gradle:common-plugin:1.1.14")
+    implementation("com.cognifide.gradle:common-plugin:1.1.15")
```

Comment:
os.arch interestingly started returning aarch64 not arm64; I covered it in common plugin to always return arm64 which is used as suffix in disp tar gz provided by adobe in sdk zip